rasa/cli/arguments/test.py

```diff
@@ -132,6 +133,22 @@ def add_test_nlu_argument_group(
         default=10,
         help="Number of cross validation folds (cross validation only).",
     )
+    comparison_arguments = parser.add_argument_group("Comparison Mode")
```

Comment:
Please also update the tests in `tests/cli`. Travis does not run them yet, but they are should fail with the changes you have done.